src/packages/excalidraw/package.json

```diff
@@ -75,10 +75,11 @@
   "scripts": {
     "gen:types": "tsc --project ../../../tsconfig-types.json",
     "build:umd": "rm -rf dist && cross-env NODE_ENV=production webpack --config webpack.prod.config.js && cross-env NODE_ENV=development webpack --config webpack.dev.config.js && yarn gen:types",
+    "build:umd:dev": "cross-env NODE_ENV=development webpack --config webpack.dev.config.js",
     "build:umd:withAnalyzer": "cross-env NODE_ENV=production ANALYZER=true webpack --config webpack.prod.config.js",
     "pack": "yarn build:umd && yarn pack",
     "start": "webpack serve --config webpack.dev-server.config.js",
-    "install:deps": "yarn install --frozen-lockfile && yarn --cwd ../../../",
+    "install:deps": "yarn --cwd ../ install --frozen-lockfile && yarn install --frozen-lockfile && yarn --cwd ../../../ install --frozen-lockfile",
```

Comment:
why is `yarn --cwd ../` needed ?

Reply:
That was left over from the npm deduping, so it's definitely not needed.  Removed in eb76191.